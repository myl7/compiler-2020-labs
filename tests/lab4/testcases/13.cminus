void main(void) {
    float a;
    a = 1 + 2.3;
    outputFloat(a);
    a = 1 * 2.3;
    outputFloat(a);
    a = 1 + 2.3 * 4;
    outputFloat(a);
    a = (1 + 2.3) * 4;
    outputFloat(a);
    return;
}
