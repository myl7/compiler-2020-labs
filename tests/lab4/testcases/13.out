3.300000
2.300000
10.200000
13.200000
